docs/developers/Building.md

```diff
@@ -77,6 +77,13 @@ See [`nix/overlays/build-tools.nix`](https://github.com/input-output-hk/cardano-
    $ cabal install --install-method=copy --installdir=/usr/local/bin
    ```
 
+7. Build without `scrypt` (for compatibility with Apple M1 chip)
+
+   ```console
+   $ cabal configure --disable-tests --disable-benchmarks -f-scrypt -O2
+   $ cabal build cardano-wallet:exe:cardano-wallet --ghc-options -O2
```

Comment:
If you already have provided `-O2` in `cabal configure` there is no point in providing it here as well?
```suggestion
   $ cabal build cardano-wallet:exe:cardano-wallet
```

It still chooses -O2:
```bash
$ cabal build cardano-wallet:exe:cardano-wallet                                                                                                                                                                                                                                      
/nix/store/pyhzxqvkbfyf4sbi35425zk8d34a16p8-cabal-install-exe-cabal-3.4.0.0/bin/cabal --project-file=/Users/anviking/IOHK/cardano-wallet/.nix-shell-cabal.project build cardano-wallet:exe:cardano-wallet
Build profile: -w ghc-8.10.7 -O2
[...]
```